packages/edit-navigation/src/store/selectors.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import { invert } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { createRegistrySelector } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { KIND, POST_TYPE, buildNavigationPostId } from './utils';
+
+/**
+ * @see resolvers.js
```

Comment:
I think it's worth repeating the doc comment for clarity and because the `@return` is different.